tests/test_block.py

```diff
@@ -31,3 +34,57 @@ def test_save_with_no_instance(self):
             self.assertTrue('Block is not associated with an instance' in
                             context.exception.args[0])
             self.assertFalse(b._put.called)
+
+        def test_load_template(self):
+            from .example_data import SimulatorFastTemplate
+            instance = lambda: None
+            instance.droplog = print
+            btype = 'SimulatorFast'
+            b = Block('sim', btype)
+            b.load_template(btype, SimulatorFastTemplate, instance)
```

Comment:
This should test the results of load_template. Check the config and template for appropriate values.